exporter/datadogexporter/translate_traces.go

```diff
@@ -208,6 +209,9 @@ func resourceSpansToDatadogSpans(rs pdata.ResourceSpans, hostname string, cfg *c
 		// TODO: allow users to configure specific spans to be marked as an analyzed spans for app analytics
 		top := getAnalyzedSpans(apiTrace.Spans)
 
+		// https://github.com/DataDog/datadog-agent/blob/443930a00bced1ea87d0067fc2f1fc3f98a48ba1/pkg/trace/agent/agent.go#L241-L243
+		traceutil.ComputeTopLevel(apiTrace.Spans)
```

Comment:
Is there any unit test we can add for this to catch it?

Reply:
well, i added back the tests that got removed here: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/3786/files#diff-0150f0fe9ebf825e7a4cebf5bdc61d443f910ae89c8b512bb4bc5f2c57dc7b56R803 . That gets us some of the way there.

I think I just missed the importance of this metric tbh